templates/index.html

```diff
@@ -21,15 +23,6 @@ <h3>{{ announcement["name"] }}</h3>
     </div>
 {% endfor %}
 <script>
-    document.querySelectorAll(".dt").forEach(function (e) {
```

Comment:
not sure what this does, did you mean to remove it?

Reply:
Yep, this is intentional.